README.md

```diff
@@ -48,6 +48,16 @@ Edit ~/.bashrc file and add $HOME/toolchain/gcc-arm-none-eabi-8-2018-q4-major/bi
 PATH=${PATH}:$HOME/toolchain/gcc-arm-none-eabi-8-2018-q4-major/bin/
 ```
 
+# Prepare python environment
+
+Some tool rely on python to run. A possible preparation is proposed below:
+
+```bash
+$ virtualenv simulation/venv
+$ source simulation/venv/bin/activate
+$ pip3 install -r simulation/requirements.txt
```

Comment:
Please mention that Python>=3.6 is required to run the simulation script.
The right way to create a virtualenv since Python 3.4 is the following:
```
$ python3 -m venv simulation/venv
$ source simulation/venv/bin/activate
$ pip install -r simulation/requirements.txt
```
This way, only python3 and python3-venv package are required.
After venv activation, python and pip version defaults to the version used to create the venv, so you can juste run 'python' and 'pip' instead of 'python3' and 'pip3'.

Reply:
Ok, good to know. This will be updated accordingly.